test.js

```diff
@@ -0,0 +1,76 @@
+/// Task №1/////////////
+
+
+const test = document.querySelector('.test')
+test.style.width = '100px'
+test.style.height = '100px'
+test.addEventListener('click', function(){
+    test.style.width = '100px'
+    test.style.height = '100px'
+    const newWidth = test.style.width
+    const newWidth2 = Number(newWidth.replace(/\D/g,''))
```

Comment:
Лишняя работа, можно сделать так

```suggestion
    const newWidth = Number(test.style.width.replace(/\D/g,''))
```